app/helpers/filter-replace.js

```diff
@@ -7,6 +7,19 @@ var filters = {
     'Research Papers in Economics': 'RePEc'
 };
 
+/**
+ * @module ember-preprints
+ * @submodule helpers
+ */
+
+/**
+ * filterReplace helper. Replaces long provider names without messing with search filter logic
+ *
+ * @class filterReplace
+ * @param {String} filter Filter
```

Comment:
Yeah, in the past I've had to declare both `class` and `method`. Otherwise, the method isn't part of anything. There may be a more elegant way.

YUIDoc writes out a file called `docs/data.json`, and the `warnings` section of that file can sometimes (but not always) help identify doc build errors.